documentation/src/main/asciidoc/topics/con_server_hinting.adoc

```diff
@@ -1,7 +1,24 @@
 [id='server-hinting_{context}']
 = Server hinting
 
-The following topology hints can be specified:
+Server hinting helps you to achieve high availability of your data in case of hardware and network failure.
+When you configure server hinting, {brandname} distributes your data across the cluster and stores backup copies of your data on as many servers, racks, and data centers as possible.
+
+NOTE: When you use replicated caches, {brandname} keeps a complete copy of all entries on each node. Server hinting distributes data across the topology so that backup copies of data are stored on different nodes in the topology than the primary owners.
+
+{brandname} uses consistent hashing mechanism to distribute data across nodes.
+The algorithm can potentially move a greater number of segments than necessary during cluster rebalancing.
+You can mitigate this by configuring a larger number of segments.
+However, using a large number of segments can lead to uneven distribution of segments.
```

Comment:
Think we can condense this with the previous sentence. In the first sentence you're basically describing how Infinispan distributes data, which is a piece of concept info that belongs outside this topic.

The algorithm is too vague. Should always use `{brandname}` in places like this where you're describing how the logic of the code works.

Also the relationship to the previous sentence is that when you configure server hinting Infinispan uses only the TopologyAwareSyncConsistentHashFactory consistent hashing algorithm. So you could probably say something like that to be more direct and clear.